CONTRIBUTING.md

```diff
@@ -73,6 +73,7 @@ We are going to set up the Firebase emulator to be able to load the data locally
 * Run the app
    * You should see your anonymous user in Authentication, and once you add new data, see it in Firestore emulator UI at: http://127.0.0.1:4000/firestore
    * If you don't see your user, delete the app from the simulator, and in the menu go to Device > Erase All Content and Settings (which resets your simulator), and try to run again
+   * If you receive the following error when you launch the emulator: _'firebase-tools no longer supports Java version before 11. Please upgrade to Java version 11 or above to continue using the emulators.'_ The openJDK install failed and you will have to install the latest JDK manually. You can download the latest version here [JDK23](https://www.oracle.com/java/technologies/downloads/#jdk23-mac)
```

Comment:
Also moved this one, and see how originally in the diff you still had quite a few changes with spacing / indentation, see screenshot, next time, you can always commit a few, push, and look at the diff to see if it's resolved (where it would show as nothing changed), vs this one shows a red space on the left compared to the right, that the spacing is off

![Screenshot 2024-10-06 at 10 28 16 AM](https://github.com/user-attachments/assets/4aa0519c-fb92-444b-93c2-f8f2697d462b)